system-apps/admin-notification-app/webClient/src/app/adminnotification-component.ts

```diff
@@ -50,12 +50,23 @@ export class AdminNotificationComponent {
 
   sendRest(title: string, message: string): void {
     let notification = new ZoweNotification(title, message, 1, "org.zowe.zlux.bootstrap")
+    let element = (<HTMLImageElement>document.getElementsByClassName('response')[0]);
+    let titleElement = (<HTMLInputElement>document.getElementsByClassName('notification-title')[0]);
+    let messageElement = (<HTMLInputElement>document.getElementsByClassName('notification-message')[0]);
+
 
     ZoweZLUX.pluginManager.loadPlugins('bootstrap').then((res: any) => {
       if (this.recipient === EVERYONE) {
         ZoweZLUX.notificationManager.serverNotify({"notification": notification, "recipient": EVERYONE})
         .then(
           (res: any) => {
+            if (res.ok) {
+              element.style.color = 'black'
+              titleElement.value = ""
+              messageElement.value = ""
+            } else {
+              element.style.color = '#b70000'
+            }
             res.json().then((json: any) => this.response = "Server Response: " + json.Response)
           },
           (error: any) => {
```

Comment:
So, somehow fetch isnt design to ever throw a promise errror:

The Promise returned from fetch() won’t reject on HTTP error status even if the response is an HTTP 404 or 500. Instead, it will resolve normally (with ok status set to false), and it will only reject on network failure or if anything prevented the request from completing.

from https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch